### DISCUSSION TITLE
Paradise support with angular 17

### DISCUSSION THREAD

**Discussion by Github743** [primefaces/community#1041] [PrimeNG Templates] [answered]
I am trying to integrate the paradise template with angular 17, but getting some issues. I brought basic license, if some one had it working on angular 17

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately Paradise will not receive an official v17 update. However, we're preparing a user guide for updating manually.

**Reply by Github743** [reply]
OK thanks for the reply how can I add slim+ menu

**Reply by Github743** [reply]
https://www.primefaces.org/layouts/rain is this available for angular
